### PR TITLE
feat: Adding migrations on separate files

### PR DESCRIPTION
**Pull request by mrestuccia**
Finally managed to run knex in this machine and separated the migration into 2 files as suggested.

### CODE REVIEW

File: server/models/migrations/20190617180248_create_pokemon.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      .notNullable()
+      .unique();
+    pokemon
+      .string('name')
+      .notNullable()
+      .unique();
+    pokemon.integer('attack').notNullable();
+    pokemon.integer('defense').notNullable();
+    pokemon
+      .enu('pokeType', [
```
Comment (aldasmallthings) [inline server/models/migrations/20190617180248_create_pokemon.ts]: you might want to access this list of enu's later for testing purposes, so you might want to consider breaking this off into a file that you can import :)

**Comment by mrestuccia** [comment]
I will add the constants separately. 
Make sense and will look cleaner.

File: server/models/migrations/20190617180248_create_pokemon.ts (hunk 1 above)
Comment (mrestuccia) [reply server/models/migrations/20190617180248_create_pokemon.ts]: OK, changed.

**Comment by aldasmallthings** [comment]
looks awesome! :)

**Pull request merged**